src/tools/crossgen2/Common/JitInterface/ThunkGenerator/gen.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+cd "$(dirname ${BASH_SOURCE[0]})"
```

Comment:
`cd /d %~dp0` is the equivalent Windows batch command. Could you add that to gen.bat?

Reply:
Done!